software/util/buildfiles.py

```diff
@@ -181,40 +202,48 @@ def exportrdf(exportType):
         allGraph.update(deloddtriples)
         currentGraph += allGraph
 
-
-        desuperseded="""PREFIX schema: <%s://schema.org/>
+        desuperseded = """PREFIX schema: <%s://schema.org/>
         DELETE {?s ?p ?o}
         WHERE{
             ?s ?p ?o;
                 schema:supersededBy ?sup.
         }""" % (protocol)
-        #Currently superseded terms are not suppressed from 'current' file dumps
-        #Whereas they are suppressed from the UI
-        #currentGraph.update(desuperseded)
+        # Currently superseded terms are not suppressed from 'current' file dumps
+        # Whereas they are suppressed from the UI
+        # currentGraph.update(desuperseded)
 
-        delattic="""PREFIX schema: <%s://schema.org/>
+        delattic = """PREFIX schema: <%s://schema.org/>
         DELETE {?s ?p ?o}
         WHERE{
             ?s ?p ?o;
                 schema:isPartOf <%s://attic.schema.org>.
-        }""" % (protocol,protocol)
+        }""" % (protocol, protocol)
         currentGraph.update(delattic)
 
-    formats =  ["json-ld", "turtle", "nt", "nquads", "rdf"]
-    extype = exportType[len("RDFExport."):]
+    formats = ["json-ld", "turtle", "nt", "nquads", "rdf"]
+    extype = exportType[len("RDFExport.") :]
```

Comment:
nit: in the [] I would not add space though.

Reply:
I'll run ruff on the file, it should fix the formatting.